tests/cli-test.py

```
@@ -0,0 +1,60 @@
+
+        # Expected results (as a dictionary for easier comparison)
+        expected_headers = {
+            'x-api-key': '12334',
+            'generic-header-1': 'generic-header-value-1',
+            'space-header': 'value-space-header'
+            # Note: generic-header2 not included as it doesn't have a colon separator
+        }
+
+        # Test exact dictionary equality
+        self.assertEqual(processed_headers, expected_headers,
+                         f"Headers don't match expected values.\nGot: {processed_headers}\nExpected: {expected_headers}")
```

Ruff (0.8.2)

Line too long (121 > 120)

(E501)